custom_components/localtuya/cover.py

```diff
@@ -105,6 +135,7 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
 
 def setup_platform(hass, config, add_devices, discovery_info=None):
     """Set up of the Tuya cover."""
+    _LOGGER.info("running setup_platform")
```

Comment:
You can remove this (at least not keep it as `info`). This goes for most of the log points which are called a lot in this file.